CHANGELOG.rst

```diff
@@ -32,6 +32,8 @@ Changelog
   :attr:`~cryptography.x509.RFC822Name.value` attribute was deprecated, users
   should use :attr:`~cryptography.x509.RFC822Name.bytes_value` to access the
   raw value.
+* Added :attr:`~cryptography.x509.CertificateRevocationList.is_signature_valid`
```

Comment:
How could you make it a property, don't you need to pass the issuers public key as an argument?